src/api/database/models/TranscriptNameMapping.ts

```diff
@@ -0,0 +1,34 @@
+import {
+    Table,
+    Column,
+    Model,
+    ForeignKey,
+    BelongsTo,
+    PrimaryKey,
+  } from 'sequelize-typescript';
+  import { UUID, STRING } from 'sequelize';
+  import User from './User';
+  import Transcript from "./Transcript";
+
+  @Table({ paranoid: true })
+  class NameMapping extends Model {
+    @PrimaryKey
+    @Column(STRING)
+    handlebarName: string;
+
+    @PrimaryKey
+    @ForeignKey(() => Transcript)
+    @Column(STRING)
+    transcriptId: string;
```

Comment:
no need for this field. name mapping should be global across all transcripts.

Reply:
I was thinking if we should add any other field to identify the users in the case of different users having the same handlebar name